src/en/space-station-14/art.md

```diff
@@ -1,5 +1,233 @@
+* When creating art for extremely fantastical or out of place things, it may be beneficial to intentionally push the style.
+Using especially vibrant colors or baking in glows into the sprite can give extra emphasis to something that _should_ look out of place.
+
+### The Future Never Looked so Old
+* While the setting is unquestionably in the future, the aesthetics of SS14 commonly and routinely reference older technology.
+The station is lit with buzzing incandescent bulbs, people send faxes to one another, and computers are still fat consoles with big glass screens.
+This retro appearance is an important factor of the visual identity of SS14: _embrace it_.
+
+
+* When creating new assets or features, playing on existing real world technology can create an intuitive understanding.
+Instead of depicting some kind of generic 'credit chit,' using paper bills is immediately recognizable.
+If you're trying to communicate the purpose of something, creating a visual association with something the player is likely familiar with can help them intuitively understand its function.
+* * If you're make a sci-fi generator, making it look similar to a real-life generator helps people understand it without close examination.
+* * Don't worry about the logistical side of retro technology.
+Any amount of ancient tech can be hand-waved away with budget restrictions and lazy scientists.
+Adhering to a personal concept of lore is secondary to serving the style.
+
+### Readability and Game Sprites
+
+* Keep the size of assets **proportional** to similar objects. 
+Obviously items will look over-sized compared to mobs, but making sure that the entire canvas isn't filled by a small object helps establish a relative scale.
+It can be confusing when an item that looks small when held appears significantly larger when dropped to the ground.
+
+
+* **Clothing** shouldn't cover excessive amounts of a mob or areas that shouldn't be covered by that region.
+Glasses shouldn't engulf an entire head and a hat shouldn't come down to the shoulders.
+Clothing should ideally have minimal overlap with other items so as to reduce visual clutter when many items are worn.
+
+
+* Mobs and direction-sensitive structures should contain **directional sprites** (directionals) for all 4 cardinal directions.
+When making directionals, the height of the asset should remain relatively constant and all details should be preserved for a consistent visual shape.
+Likewise, the color and placement of shadows should also shift to better depict the item turning in 3D space.
+
+## Technique
+
+### Perspective
+
+* **Structures** in the game world should be in a 3/4 perspective.
+This includes various machines, furniture, and decor.
+A 3/4's perspective means that they are angled straight forward with the viewer looking down on them from above. 
+This gives you a clear view of the top as well as the front of the object.
+
+
+* **Mobs**, **clothing**, and **items** are rendered in a flat perspective. 
```

Comment:
This section implies that items should never be 3/4ths, which is simply not true. Items like boxes and power cells need the definition to work. There should be some kind of addendum to this mentioning that.